cmake/treedata/tvos/subdirs.txt

```diff
@@ -0,0 +1,17 @@
+xbmc/platform/linux                   platform/linux
+xbmc/platform/linux/network           platform/linux/network
+xbmc/input/touch                      input/touch
+xbmc/input/touch/generic              input/touch/generic
+xbmc/platform/posix                   posix
+xbmc/platform/posix/filesystem        platform/posix/filesystem
+xbmc/platform/posix/utils             platform/posix/utils
+xbmc/platform/darwin                  platform/darwin
+xbmc/platform/darwin/tvos             platform/tvos
+xbmc/platform/darwin/ios-common       platform/ios-common
+xbmc/platform/darwin/osx/network      platform/osx/network
+xbmc/platform/darwin/osx/peripherals  platform/osx/peripherals
```

Comment:
tvOS isn't allowed to use osx code. If it is need it has to be moved into a directory named somthing like darwin.